stack-8.4.4.yaml

```diff
@@ -1,4 +1,7 @@
 resolver: lts-12.26
 
-ghc-options:
-  "$locals": -fhide-source-paths
+extra-deps:
+- Cabal-2.4.1.0@rev:0
```

Comment:
Seems like this is not enough. The custom `Setup.hs` file should be present in order to support older GHC...

Reply:
This is really strange because I built locally on my machine with GHC-8.4.4 and everything worked... But I will try to add `Setup.hs` file :+1:

Reply:
Heh...